src/main/java/org/spongepowered/common/block/entity/SpongeBlockEntityArchetypeBuilder.java

```diff
@@ -99,7 +102,7 @@ public BlockEntityArchetype.Builder from(final BlockEntityArchetype value) {
     public BlockEntityArchetype.Builder state(final BlockState state) {
         final net.minecraft.world.level.block.state.BlockState blockState = (net.minecraft.world.level.block.state.BlockState) state;
         if (!((BlockStateBridge) (blockState)).bridge$hasTileEntity()) {
-            new IllegalArgumentException("BlockState: "+ state + " does not provide TileEntities!").printStackTrace();
+            SpongeBlockEntityArchetypeBuilder.LOGGER.warn("BlockState " + state + " does not provide TileEntities!", new IllegalArgumentException());
```

Comment:
TileEntities -> BlockEntities

Reply:
you'll also want to replace string concat with string interpolation

```suggestion
            SpongeBlockEntityArchetypeBuilder.LOGGER.warn("BlockState {} does not provide TileEntities!", state, new IllegalArgumentException());
```